app/src/test/java/org/simple/clinic/summary/linkId/LinkIdWithPatientUpdateTest.kt

```diff
@@ -60,13 +60,12 @@ class LinkIdWithPatientUpdateTest {
 
   @Test
   fun `when add identifier is clicked, then update UI`() {
-    val patientFetchedModel = defaultModel.linkIdWithPatientViewShown(patientUuid, identifier)
     updateSpec
-        .given(patientFetchedModel)
+        .given(defaultModel)
```

Comment:
Shouldn't this model be the patient fetched model instead of the `defaultModel`. We might have used the wrong model when adding this screen initially, but I think we can correct this now.

Reply:
Ah right, Good catch. Will fix that 👍🏾